iOS_SDK/OneSignalSDK/build_fat_framework.sh

```diff
@@ -79,6 +79,11 @@ RELEASE_OUTPUT_FRAMEWORK_DIR="${WORKING_DIR}/Framework/OneSignal.framework"
 rm -rf "${WORKING_DIR}/Framework/OneSignal.framework"
 cp -a "${FINAL_FRAMEWORK}" "${WORKING_DIR}/Framework/OneSignal.framework"
 
+echo "Copying public header for SwiftPM"
+#Copy the public header to the SwiftPM public headers directory
+rm -rf "${WORKING_DIR}/SwiftPM/Public/Headers/OneSignal.h"
```

Comment:
Let's remove the whole SwiftPM folder here.
```sh
rm -rf "${WORKING_DIR}/SwiftPM"
```

That way if we remove or move files around in this directory in future builds we don't have old file carry over.

Reply:
fixed!

Reply:
👍